src/quiz.js

```diff
@@ -7,18 +7,20 @@ import UserService from "./service/UserService";
 
 const quizUi = new QuizUI(document);
 const wordService = new WordService();
-const userId = localStorage.getItem(CURR_USER_KEY);
+const userId = JSON.parse(localStorage.getItem(CURR_USER_KEY));
 const userService = new UserService();
+let wordsLeft = 0;
```

Comment:
initialize `wordsLeft` as `-1` to distinguish it from the case that the user really completed today's words and has `wordsLeft == 0`